### DISCUSSION TITLE
[QUESTION]: 单条日志4M无法正常发送

### DISCUSSION THREAD

**Discussion by denghanling** [alibaba/ilogtail#2136] [Help]
采集配置：
enable: true
inputs:
  - Type: file_log
    LogPath: /ilogtail-code/kafkaLog/
    FilePattern: '*.log'
flushers:
  - Type: flusher_stdout
    OnlyStdout: true

启动参数配置：
{   
    "buffer_file_num" : 100,
    "buffer_file_size" : 83886080,
     "max_read_buffer_size": 8388608,
     "check_point_dump_interval": 10
}

单条日志4M
![image](https://github.com/alibaba/ilogtail/assets/35355143/58c61e2a-6eb9-4794-ac1c-fd7dd197db3f)

stdout无打印日志
![image](https://github.com/alibaba/ilogtail/assets/35355143/617cb2a4-54d7-44f9-878a-42c948e082ca)

一开始测的是发kafka，发现无法正常发送kafka，所以改成stdout，发现也发不了，大概是不是input哪里问题，是代码问题还是配置问题？

**Comment by Takuka0311** [comment]
有一些相关日志信息吗？
